RISC-V support

When running auditwheel on linux_riscv64 wheel it crashes.

```
[189:24] Fixing wheelhouse/cryptography-38.0.4-cp311-cp311-linux_riscv64.whl
[189:24] AssertionError: [{'name': 'linux_riscv64', 'aliases': [], 'priority': 0, 'symbol_versions': {}, 'lib_whitelist': [], 'blacklist': {}}]
[189:26] Traceback (most recent call last):
[189:26] File "/builds/funkwhale/funkwhale/tools/bin/auditwheel", line 5, in <module>
[189:26] from auditwheel.main import main
[189:26] File "/builds/funkwhale/funkwhale/tools/lib/python3.11/site-packages/auditwheel/main.py", line 16, in <module>
[189:26] from . import main_addtag, main_lddtree, main_repair, main_show
[189:26] File "/builds/funkwhale/funkwhale/tools/lib/python3.11/site-packages/auditwheel/main_repair.py", line 7, in <module>
[189:26] from .policy import (
[189:26] File "/builds/funkwhale/funkwhale/tools/lib/python3.11/site-packages/auditwheel/policy/__init__.py", line 118, in <module>
```

It looks like policies for musllinux and manylinux are missing.

--- a/src/auditwheel/policy/__init__.py
+++ b/src/auditwheel/policy/__init__.py
@@ -286,6 +286,7 @@ def _fixup_musl_libc_soname(libc: Libc, arch: str, whitelist):
             "s390x": "libc.musl-s390x.so.1",
             "ppc64le": "libc.musl-ppc64le.so.1",
             "armv7l": "libc.musl-armv7.so.1",
+            "riscv64": "libc.musl-riscv64.so.1",
         }
     }
     new_whitelist = []

--- a/src/auditwheel/policy/musllinux-policy.json
+++ b/src/auditwheel/policy/musllinux-policy.json
@@ -21,6 +21,8 @@
       "s390x": {
       },
       "armv7l": {
+      },
+      "riscv64": {
       }
     },
     "lib_whitelist": ["libc.so", "libz.so.1"],
@@ -42,6 +44,8 @@
       "s390x": {
       },
       "armv7l": {
+      },
+      "riscv64": {
       }
     },
     "lib_whitelist": ["libc.so", "libz.so.1"],

